lib/watchpack.js

```diff
@@ -24,18 +24,20 @@ function addWatchersToSet(watchers, set) {
 	}
 }
 
-const stringToRegexp = ignored => {
-	const source = globToRegExp(ignored, { globstar: true, extended: true })
-		.source;
+const stringOrRegexpToRegexp = ignored => {
+	if (ignored instanceof RegExp) {
+		return ignored.source;
```

Comment:
could you add a check for `ignored.flags === ""` and throw an Error if not?

Otherwise we would handle that incorrectly.

Reply:
good catch - updated and added test